ERCS/erc-7540.md

```diff
@@ -41,6 +41,7 @@ The existing definitions from [ERC-4626](./eip-4626.md) apply. In addition, this
 - asynchronous deposit Vault: a Vault that implements asynchronous Requests for deposit flows
 - asynchronous redemption Vault: a Vault that implements asynchronous Requests for redemption flows
 - fully asynchronous Vault: a Vault that implements asynchronous Requests for both deposit and redemption flows
+- requester: owner of the Request, who can manage any actions related to the Request including claiming the `assets` or `shares`
```

Comment:
Would capitalise all definitions, also should we add the `Operator` here?

Reply:
I would not capitalise just because in 4626 the owner/receiver aren't capitalised either, nor are assets and shares. I can't think of a particularly strong reason for that, but would keep it consistent now.

I agree we should add operator though!

Reply:
@0xfarhaan https://github.com/ERC4626-Alliance/ERCs/pull/32